### ISSUE TITLE
Cannot change to intel

### ISSUE THREAD

**Issue by moedn** [paltas/FedoraPrime#23]
First: Thanks a lot for the effort to program this piece of software. I'm absolutely in need of this and I don't have the skills to do it by myself. Thank you.

**The background**

 - HP ZBook 15v G5 (GPU: Intel + Nvidia Quadro P600)
 - nouveau causes a very unstable system → blacklisted nouveau and installed proprietary Nvidia drivers instead (rpmfusion-nonfree-nvidia-driver)
 - Driver works great, super stable system

**The issue**

 - the external GPU (Nvidia Quadro P600) is always on and quickly drains my battery
 - I rarely need the external GPU (only when working with CAD), but when I need it, I depend on it

**→** I would like to disable the Nvidia card and enable it again, when I need to.

I installed the package as described, then I do:

`sudo fedora-prime-select intel`
which gives:
> Successfully switched to intel.
> Please logout for changes to take effect.

Logging out & in or rebooting won't change anything:
`glxinfo | grep 'OpenGL renderer string'`
gives
> OpenGL renderer string: Quadro P600/PCIe/SSE2

**Comment by xEt3** [comment]
I have the same problem, could you solve this?? @moedn

**Comment by moedn** [comment]
@xEt3 nope, sorry, didn't work on it, yet. So I have my nvidia GPU all time on and live with short battery life.
Instead I found [this](https://www.reddit.com/r/Fedora/comments/8sqosq/how_to_disable_discrete_nvidia_gpu/) lately. I may set up a test system and try it out. Nouveau caused a _very_ unstable system in my case. I need my laptop every day. Never touch a running system :)

**Comment by moedn** [comment]
@xEt3 just found this fork: https://github.com/wildtruc/nvidia-prime-select
Seems to be active and up to date. FedoraPrime has only been tested for Fedora 22 & 23. As I'm on 30, this might be the source of the issue described above.
